lbrynet/daemon/Daemon.py

```diff
@@ -2097,18 +2097,21 @@ def jsonrpc_claim_list_by_channel(self, page=0, page_size=10, uri=None, uris=[])
         defer.returnValue(response)
 
     @AuthJSONRPCServer.auth_required
-    def jsonrpc_transaction_list(self):
+    def jsonrpc_transaction_list(self, include_tip_info=False):
         """
         List transactions belonging to wallet
 
         Usage:
-            transaction_list
+            transaction_list [--include_tip_info]
+
+        Options:
+            --include_tip_info  :(bool), False
 
         Returns:
             (list) List of transactions
```

Comment:
You should expand on how these results are formatted, and include the behavior of `is_tip` depending on the `include_tip_info` argument.